frontend/src/model/releases/ReleaseForm.tsx

```diff
@@ -78,6 +93,42 @@ export default function ReleaseForm({
     </Typography>
   )
 
+  const fileProgressText = () => {
+    if (!currentFileUploadProgress) {
+      return <Typography>Could not determine file progress</Typography>
+    }
+    if (uploadedFiles && uploadedFiles.length === filesToUploadCount) {
+      return <Typography>All files uploaded successfully.</Typography>
+    }
+    return currentFileUploadProgress.uploadProgress < 100 ? (
+      <Grid container direction='row' spacing={1}>
+        <Grid item>
+          <Typography fontWeight='bold'>
+            [File {uploadedFiles ? uploadedFiles.length + 1 : '1'} / {filesToUploadCount}] -
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Typography fontWeight='bold'>{currentFileUploadProgress.fileName}</Typography>
+        </Grid>
+        <Grid item>
+          <Typography>uploading {currentFileUploadProgress.uploadProgress}%</Typography>
```

Comment:
The starting capitalisation seems random here.  'All files' vs 'uploading' vs 'File 1' vs 'received - waiting for response'.  We should probably try to use the same everywhere.

Reply:
I think these are ok if you read them as full sentences, so it would read:

"[File 1/2] - myFile.txt uploading 55%"

vs

"All files uploaded successfully."

It looks a bit more confusing in-code because they're split up by `<Grid />`s